api/app.py

```diff
@@ -26,57 +27,31 @@ def get_featured_products():
     # Hardcoded. Replace with actual data from DB
     # select 4 random products from product table
     # return them as a list
-    response = jsonify({
-        "products": [
-            {
-                "id": 1,
-                "name": "Amazon Basics USB Wired Computer Keyboard and Wired Mouse Bundle Pack",
-                "price": 2599,
-                "category": "accessories",
-                "stock": 41,
-                "description": "<ul><li><span> Low-profile keys provide a quiet, comfortable typing experience  </span></li>",
-                "image": "https://m.media-amazon.com/images/I/71y-jMVFfTL._AC_SL1500_.jpg",
-                "alt_images": [
-                    "https://m.media-amazon.com/images/I/51o3dhWxLSL._AC_SL1500_.jpg",
-                    "https://m.media-amazon.com/images/I/51P4u8QBpJL._AC_SL1367_.jpg",
-                    "https://m.media-amazon.com/images/I/81zDsqaBwAL._AC_SL1500_.jpg",
-                    "https://m.media-amazon.com/images/I/91K61OKfPkL._AC_SL1500_.jpg"
-                ],
-                "reviews": [
-                    {
-                        "username": "Maxime1",
-                        "rating": 5,
-                        "review": "If you are into basics, I don't think you can get a better deal than this."
-                    }
-                ]
-            },
-            {
-                "id": 2,
-                "name": "Wireless Keyboard and Mouse Combo",
-                "price": 4299,
-                "category": "accessories",
-                "stock": 60,
-                "description": "<ul><li><span> ?Ergonomic Wireless Keyboard and Mouse? JOYACCESS wireless keyboard and mouse combo designed ergonomic</li></ul>",
-                "image": "https://m.media-amazon.com/images/I/71fH1bumcBL._AC_SL1500_.jpg",
-                "alt_images": [
-                    "https://m.media-amazon.com/images/I/71xCKnpg08L._AC_SL1500_.jpg",
-                    "https://m.media-amazon.com/images/I/71CYi3hoZKL._AC_SL1500_.jpg",
-                    "https://m.media-amazon.com/images/I/61mpN2N5yeL._AC_SL1500_.jpg",
-                    "https://m.media-amazon.com/images/I/712-hRbokcL._AC_SL1500_.jpg",
-                    "https://m.media-amazon.com/images/I/71y-DSs04JL._AC_SL1500_.jpg",
-                    "https://m.media-amazon.com/images/I/71zrUEKoX2L._AC_SL1500_.jpg"
-                ],
-                "reviews": [
-                    {
-                        "username": "Frieda.Krajcik72",
-                        "rating": 4,
-                        "review": "Super clavier!"
-                    }
-                ]
-            }
-        ]
-    })
-
+    
+    c_featuredProducts = db.cursor(dictionary=True)
+    
+    c_featuredProducts.execute('SELECT details FROM product LIMIT 4')
```

Comment:
Can you try this: 
```suggestion
    c_featuredProducts.execute('SELECT details FROM product ORDER  BY RAND() LIMIT 4')
```
I just googled and found this. If it gives you 4 random rows every time that's great.

Btw it also seems this is a very inefficient query. But only if we have a large number of rows. We just have 500-ish so it should be perfectly fine for us. Simple too